### ISSUE TITLE
No appNewVersion for Thunderbird, causes continued update prompt

### ISSUE THREAD

**Issue by rs1278** [Installomator/Installomator#1105]
Can be fixed by revising with the following code:

thunderbird)
    name="Thunderbird"
    type="dmg"
    downloadURL="https://download.mozilla.org/?product=thunderbird-latest&os=osx&lang=en-US"
    appNewVersion="$(curl -fs "https://download.mozilla.org/?product=thunderbird-latest&os=osx&lang=en-US" | grep -i "releases" | sed -e 's/.*releases\/\(.*\)\/mac.*/\1/')"
    expectedTeamID="43AQ936H96"
    blockingProcesses=( thunderbird )
    ;;

**Comment by acodega** [comment]
Would you like to open a PR? We have a guide at https://github.com/Installomator/Installomator/wiki/GitHub-howto-create-PRs

**Comment by rs1278** [comment]
Yes, thank you. I did not see that guide.



Roman Sammartino
Systems Administrator - NOAA N-Wave
Contractor - Cherokee Nation System Solutions,  LLC
***@***.***
Google Voice (forwards to cell):
‪(505) 398-0416‬

**Comment by rs1278** [comment]
yes, but I believe i need write access to the installomator repo

**Comment by duderin0** [comment]
Since its been 2 weeks, I've submitted my own (#1128 ) and have used XPath instead of the curl options, since I couldn't get the above to work.
